modules/sigmaExporter/src/main/java/uk/ac/ox/oii/sigmaexporter/ZipHandler.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package uk.ac.ox.oii.sigmaexporter;
+
+import java.io.*;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.StandardCopyOption;
```

Comment:
Many unnecessary imports and commented code here. At least I would remove unused imports in this and other files.